README.md

```diff
@@ -343,10 +343,11 @@ requires:
   organizations: ["org1", "org2"]
   teams: ["org1/team1", "org2/team2"]
 
-  # A user must have a permission in this list for their approval to count for
-  # this rule. Valid permissions are "admin", "maintain", "write", "triage",
-  # and "read".
-  permissions: ["admin", "maintain", "write"]
+  # A user must have at least the minimum permission in this list for their
+  # approval to count for this rule. Valid permissions are "admin", "maintain",
+  # "write", "triage", and "read". Specifying more than one permission is
+  # useful to control reviewer assignment.
```

Comment:
It might be helpful to clarify how specifying more than one permission affects reviewer assignment? Perhaps mentioning that there are more details below.